prac_03/Do this now.py

```diff
@@ -12,7 +12,12 @@
 # print(things[-2])
 
 
-names = ["Bob", "jimbo", "sally"]
-for i in range(len(names)):
-    names[i] = names[i].title()
-print(names)
\ No newline at end of file
+# names = ["Bob", "jimbo", "sally"]
+# for i in range(len(names)):
+#     names[i] = names[i].title()
+# print(names)
+
+def fn(x, y):
+    z = x + y
+
+print(fn(1, 2))
```

Comment:
Always remember to add a blank line at the end, it clears up that PEP8 underline